jetty-jakarta-servlet-api/pom.xml

```diff
@@ -314,5 +327,15 @@
         <enabled>true</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>jakarta.staging</id>
+      <url>https://jakarta.oss.sonatype.org/content/groups/staging</url>
```

Comment:
FYI, the staged jakarta artifacts can be found at this repo.

Reply:
Can we add TODOs on things that should be removed once the real release is available.

Reply:
@gregw done.